File: testcases/paper/Array alias/Array.py
class Array:
    i = [0,0,0,0,0]
    k = i

    def method1(self):
        self.i[1] = 1

    def method2(self):
        self.i[2] = 2

    def method3(self):
        self.i[3] = 3

    def aliasMethod(self):
        self.k[1] = 3

# Alias doesn't count: LCOM4 = 2 or 4
# Alias counts:        LCOM4 = 1 or 3

File: testcases/paper/Array alias/Array.cpp
class Array {
    int i [50];
    int(&j)[50] = i;

    void method1() {
        i[1] = 1;
    }

    void method2() {
        i[2] = 2;
    }

    void method3() {
        i[3] = 3;
    }

    void aliasMethod() {
        j[1] = 3; 
    }
};

// Alias doesn't count: LCOM4 = 2 or 4
// Alias counts:        LCOM4 = 1 or 3